pkgs/top-level/all-packages.nix

```diff
@@ -31395,7 +31395,10 @@ with pkgs;
 
   socialscan = with python3.pkgs; toPythonApplication socialscan;
 
-  sonic-lineup = libsForQt5.callPackage ../applications/audio/sonic-lineup { stdenv = gcc10StdenvCompat; };
+  sonic-lineup = libsForQt5.callPackage ../applications/audio/sonic-lineup {
+    bzip2 = bzip2_1_1;
+    stdenv = gcc10StdenvCompat;
```

Comment:
Maybe the error is caused by this?

Reply:
If I comment that, error becomes:

> match/src/FullDTW.h:86:17: error: 'size_t' was not declared in this scope; did you mean 'std::size_t'?

Reply:
~~If both variants don't work I'd rather remove this again to not introduce another error.~~

It was there before, oh well